libqtile/widget/net.py

```diff
@@ -41,22 +44,24 @@ def __init__(self, **config):
         self.add_defaults(Net.defaults)
         if not isinstance(self.interface, list):
             if self.interface is None:
-                self.interface = ["all"]
+                self.interface = ['all']
             elif isinstance(self.interface, str):
                 self.interface = [self.interface]
             else:
-                raise AttributeError("Invalid Argument passed: %s\nAllowed Types: List, String, None" % self.interface)
+                raise AttributeError(
+                    (f'Invalid Argument passed: {self.interface}\n'
+                     f'Allowed Types: List, String, None'))
         self.stats = self.get_stats()
 
     def convert_b(self, num_bytes: float) -> Tuple[float, str]:
         """Converts the number of bytes to the correct unit"""
         factor = 1000.0
 
         if self.use_bits:
-            letters = ["b", "kb", "Mb", "Gb", "Tb", "Pb", "Eb", "Zb", "Yb"]
+            letters = ['b', 'kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Eb', 'Zb', 'Yb']
             num_bytes *= 8
         else:
-            letters = ["B", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"]
+            letters = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
```

Comment:
Let's not pollute your commit with such changes, and rather try to keep it atomic. If you really care about the codebase being consistent on single and double quotes, please do the change across the whole code and open another PR for this.